.gitignore

```diff
@@ -2,6 +2,7 @@
 node_modules
 npm-debug.log
 .npm
+.env.local
```

Comment:
You need two environment variables:
- `NEXT_PUBLIC_ORAMA_ENDPOINT`
- `NEXT_PUBLIC_ORAMA_API_KEY`

these are both public data, I didn't want to hardcode them in the codebase. Also, we have two different values (a couple for development/qa and a couple for production).

How would you suggest to proceed?

Reply:
No what I am saying os the default value should be an empty string. But no direct process.env access should be done by other places of the code.

Reply:
not sure I follow how to share these variables then?

Reply:
You shouldnt share these values. Nor should people be using .env files. You can simply expose them to your environment by any other means.

If you want to use dot env files then we need to add dotenv support to this repository and I don't think that makes sense atm.

What I am referring is that within your code you're accessing process.env.directly (inside next.orama.mjs) my suggestion was simply move those calls to next.constants.mjs and if there is no given env variable fallback to an empty string so that the variable wont become undefined

Reply:
Alright, I already moved these accesses to `process.env` into the constants file. I will set an empty string as a fallback value, but the search will crash when these variables are not set

Reply:
That's alright. I assume search would only crash on QA/Prod right? Because for local development servers (npm run serve) it should use local data, right? Or at least thats what we chatted

Reply:
Like Id go to lengths to say that if these variables are not defined then it should use local database index instead of cloud)

Reply:
Not yet. Right now it's relying on Orama Cloud for local development, too. Setting up the local server is doable but quite expensive in terms of effort and I'd probably do it in a separate PR